mod_reforged/hooks/config/strings.nut

```diff
@@ -992,7 +999,7 @@ foreach (vanillaDesc in vanillaDescriptions)
 		Effects = [{
 			Type = ::UPD.EffectType.Passive,
 			Description = [
-				"Attacks against [surrounded|Concept.Surrounding] targets deal " + ::MSU.Text.colorPositive("10%") + " more damage and have " + ::MSU.Text.colorPositive("+10%") + " armor penetration per character surrounding the target."
+				"Attacks against [surrounded|Concept.Surrounding] targets deal " + ::MSU.Text.colorPositive("10%") + " more damage and have " + ::MSU.Text.colorPositive("+10%") + " armor penetration per ally surrounding the target."
```

Comment:
`character` is correct because surrounding in vanilla works even if a character of a faction hostile to you is surrounding the target as long as he is also hostile to your target.